### DISCUSSION TITLE
Opening Balance in a period is incorporating transactions

### DISCUSSION THREAD

**Discussion by focher** [firefly-iii/firefly-iii#8974] [Firefly III]
My expectation is that the opening balance on an account in any given period does **not** incorporate any transactions on that first day. However, it clearly is. Here's an example:

1. The true opening balance on June 1st is $10,000.
2. There is a withdrawal transaction on June 1st of $1,000.
3. The Opening Balance in Firefly shows the opening balance is $9,000.

This happens whether those transaction are deposits, withdrawals, or transfers. Typically in accounting, the opening balance should not include any same day transactions.

Before I report this is a bug, I wanted to see if others experience this issue and also gauge opinions whether I should ask for this to be changed.

**Comment by JC5** [comment]
To mimic the behavior you seek, set the opening balance to be the day before, i.e. May 31st.
